cosmos/bouachain.json

```diff
@@ -0,0 +1,53 @@
+{
+  "rpc": "https://rpc.bouachain.com/",
+  "rest": "https://lcd.bouachain.com/",
+  "chainId": "bouachain",
+  "chainName": "bouachain",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/bouachain/chain.png",
+  "walletUrl": "https://wallet.keplr.app/chains/bouachain",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/bouachain",
```

Comment:
Hi! Could you please remove this two lines? The type of page is not supported by default.

Reply:
Line 7 and 8 have been removed.